### ISSUE TITLE
Gating away from Crypt Closing

### ISSUE THREAD

**Issue by cambragol** [cambragol/advent-of-the-trinity#271]
**Describe the bug**
After sealing an artifact, and being teleported to the surface, if the player gates away, the crypt is closed at the location that the player gated to.

I have code to handle stepping or blinking away, but not gating.

![AVATAR EXE 2021-08-13 at 1 34 35 PM](https://user-images.githubusercontent.com/5019739/129305475-c525ceed-6fcb-4d76-bf93-e5c98bc9dadc.png)

**Comment by cambragol** [comment]
I have fixed this for 'Gate' and for the Wand, which should be the last two ways that a player can move away from a spot.

I may need to revisit the 'Crypt Collapse' animation however. Currrently the player is teleported to the entrance of the Crypt after sealing an artifact. Then, when they move away, the crypt collapses.

Better, might be to have them appear one space to the south of the Crypt, then have the crypt collapse immediately.

I am not sure why I set it up the other way....
